Add a product behavior section in docs (#4952)
alcionai/corso: website/docs/support/known-issues.md

* Groups and Teams support is available in an early-access status, and may be subject to breaking changes.

* Restoring the data into a different Group from the one it was backed up from isn't currently supported

> * Multipart emails with both html and text version will only contain the html version in export.

Reviewer: ```suggestion
* Exports of multipart emails containing both html and text versions will only produce the html version of the email.
```
